interpreter/test/start.wast

```diff
@@ -87,6 +89,11 @@
   (start $main)
 )
 
+(module
+  (func $print (import "spectest" "print"))
+  (start $print)
+)
+
```

Comment:
OK gotcha.

My point still stands though: we need to have a test where an import's signature is declared as something else than `void ()`. That test needs to fail.